2-ui/5-loading/02-script-async-defer/article.md

```diff
@@ -31,7 +31,7 @@ There are some workarounds to that. For instance, we can put a script at the bot
 
 But this solution is far from perfect. For example, the browser notices the script (and can start downloading it) only after it downloaded the full HTML document. For long HTML documents, that may be a noticeable delay.
 
-Such things are invisible for people using very fast connections, but many people in the world still have slower internet speeds and use far-from-perfect mobile internet.
+Such things are invisible for people using very fast connections, but many people in the world still have slow internet speeds and use far-from-perfect mobile internet.
```

Comment:
I think this is OK either way. More important to me would be adding changing the last phrase to " use **a** far-from-perfect mobile internet **connection**". Unless you want "internet" to be synonymous with "internet connection".